### PR TITLE
Create the mobile website section to list the portfolio projects

### PR DESCRIPTION
**Pull request by d4nQw3rty**
*Create the main container using CSS Grid to contain all the cards
*Use Flexbox to lay out an individual card.
*Implement the button interactions (enable, hover, etc).
*Build About myself section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,39 +7,185 @@
...
+                          </ul>
+                      <p class="card-work-text">A daily selection
+                         of privately personalized reads; no accounts or sign-ups required.
+                      </p>         
+                      <ul class="tags">
+                          <li class="tag-html"><p>html</p></li>
+                          <li class="tag-css"><P>css</P></li>
+                          <li class="tag-js"><p>javaScript</p></li>
+                      </ul>
+                       <button type="button" class="button-enable button-enable-text" >See Project</button>
```
Comment (vechicin) [inline index.html]: Your buttons don't have the required interaction stylings. Please add them to all.

Screen Shot 2022-07-07 at 12 06 07

File: index.html [hunk 2]
```diff
@@ -7,39 +7,185 @@
...
+             </div>
+             <ul class="social-media-icons">
+                  <li><a href="https://twitter.com/Danys_Linares" target="_blank" rel="noopener noreferer"><img src="img/Vector-twitter.svg" alt="Twitter link"></a></li>
+                  <li><a href="https://www.linkedin.com/in/danys-linares-6a328b238?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BnkyI5IMjTzSg4PVJIZh%2BMw%3D%3D" rel="noopener noreferer"><img src="img/Linkedin-icon.svg" alt="Linkedin link"></a></li>
+                  <li><a href="https://medium.com/" target="_blank" rel="noopener noreferer" ><img src="img/Vector-M.svg" alt="Medium link"></a></li>
+                  <li><a href="https://github.com/d4nQw3rty" target="_blank" rel="noopener noreferrer"><img src="img/Vector-github.svg" alt="Github link"></a></li>
+                  <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+              </ul>                                  
+         </section>    
+         <section id="main-container">
```
Comment (vechicin) [inline index.html:41]: This section doesn't have the margins defined in the Figma template. Please fix them.

Screen Shot 2022-07-07 at 12 07 13

Screen Shot 2022-07-07 at 12 07 31

File: index.html [hunk 3]
```diff
@@ -7,39 +7,185 @@
...
+               </div> 
+           </section>           
+           <section id="about-myself">
+                <h2 class="Primary-Text">About<br>
+                    Myself
+                </h2>
+                <p class="Primary-Content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>          
+                <div class="Social-Media">
+                    <p class="Lets-Connect">LET’S CONNECT</p>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in an `<hx>`.

Screen Shot 2022-07-07 at 12 23 15

File: index.html [hunk 4]
```diff
@@ -7,39 +7,185 @@
...
+                <div class="Social-Media">
+                    <p class="Lets-Connect">LET’S CONNECT</p>            
+                </div>
+                <ul class="social-media-icons">
+                     <li><a href="https://twitter.com/Danys_Linares" target="_blank" rel="noopener noreferer"><img src="img/Vector-twitter.svg" alt="Twitter link"></a></li>
+                     <li><a href="https://www.linkedin.com/in/danys-linares-6a328b238?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BnkyI5IMjTzSg4PVJIZh%2BMw%3D%3D" rel="noopener noreferer"><img src="img/Linkedin-icon.svg" alt="Linkedin link"></a></li>
+                     <li><a href="https://medium.com/" target="_blank" rel="noopener noreferer" ><img src="img/Vector-M.svg" alt="Medium link"></a></li>
+                     <li><a href="https://github.com/d4nQw3rty" target="_blank" rel="noopener noreferrer"><img src="img/Vector-github.svg" alt="Github link"></a></li>
+                     <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+                 </ul>
```
Comment (vechicin) [inline index.html]: The order of the icons doesn't match that of the template design. Please take a look at the Figma template and fix it.

File: index.html [hunk 5]
```diff
@@ -7,39 +7,185 @@
...
+                 <div id="about-myself-button-holder">
+                      <button type="button" class="button-enable button-enable-text">Get my resume</button>
```
Comment (vechicin) [inline index.html]: Your button doesn't have interaction styling. Please take a look at the previous comment and fix it.

File: style.css [hunk 6]
```diff
@@ -190,3 +193,620 @@
...
+  width: 100%;
+  height: 100%;
+}
+
+.project-title {
+  /* Project Title */
+
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```
Comment (vechicin) [inline style.css:249]: None of your `font-weight` rules are working. That is most likely caused by not importing the correct font weight from Google Fonts. Please refer to [this video](https://www.youtube.com/watch?v=mRoBBOTYNz4&t=184s) to fix this issue.

Screen Shot 2022-07-07 at 13 03 24

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @d4nQw3rty ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @d4nQw3rty , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**